poller/README.md

```diff
@@ -102,6 +105,7 @@ Key                      | Default Value  | Description
 `minNodes` (DEPRECATED)  | 1              | DEPRECATED: Minimum number of Cloud Spanner nodes that the instance can be scaled IN to.
 `maxNodes` (DEPRECATED)  | 3              | DEPRECATED: Maximum number of Cloud Spanner nodes that the instance can be scaled OUT to.
 `scalerURL`              | `http://scaler`| URL where the scaler service receives HTTP requests.
+`linearRoundingPrecision`| 1000           | Number used for suggested size comparison that rounds up by 100 PU if suggestedSize is less than this value, or rounds up by 1000 PU if suggestedSize is greater than this value. Only used when scaling with the `LINEAR` method.
```

Comment:
Scaling increments of 100 processing units is supported only until instance reaches the capacity of 1000 processing units. Above that, scaling in increments of 100 processing units is not supported by the product. It is only possible to consume full nodes for instances greater than or equal to 1 node. There are no current plans to change this.

Reply:
Hi Ben, thanks for the PR. 

However as  Shambhu has pointed out, scaling in 100 PU increments in only supported by Cloud Spanner instances with less than 1000 PUs. Above that, [scaling is supported only in 1000 PUs increments](https://cloud.google.com/spanner/docs/compute-capacity#compute_capacity).

[Line 43 in scaler/scaler-core/utils.js](https://github.com/cloudspannerecosystem/autoscaler/blob/9f2a3fd81420258eef437b228a719f038bb3e589/scaler/scaler-core/utils.js#L43) is actually working as intended to reflect the product capabilities.

Therefore, I will close this pull request.